frontend/javascripts/libs/input.js

```diff
@@ -308,6 +308,8 @@ export class InputMouse {
   position: ?Point2 = null;
   triggeredByTouch: boolean = false;
   delegatedEvents: { string?: Function };
+  isDragging: boolean;
```

Comment:
If you define `isDragging` as a local variable within this module (instead of class), that state will be shared across all `InputMouse` instances. I think, this should solve the problem with "drag into other plane and scroll there" :)

Reply:
@philippotto Your suggested fix works perfectly fine :+1: 
You can give this PR another review.